src/ucp/wireup/select.c

```diff
@@ -502,6 +505,7 @@ static UCS_F_NOINLINE ucs_status_t ucp_wireup_select_transport(
         }
 
         is_reachable = 0;
+
```

Comment:
can we revert the changes in this function? seems cosmetic/leftover

Reply:
not all of them, i modify `calc_score` params. others are quite trivial, remove anyway?

Reply:
ok